modules/pel/peltool/peltool.py

```diff
@@ -541,11 +541,30 @@ def printPELCount(path: str, config: Config, extension: str):
     print("{\n    \"Number of PELs found\": "+str(count)+"\n}")
 
 
+class CustomFormatter(argparse.HelpFormatter):
+"""
+This class is to enhance the formatting of argparse help messages 
+for arguments that have choices and allow multiple values.
+"""
+    def _format_action(self, action):
+        # Override the default formatting for choices when nargs='+' is used
+        if action.choices and action.nargs == '+':
```

Comment:
So the custom format applies to choices when using nargs, and for other data types, it defaults to the standard format?

Reply:
Yeah, custom format will apply only if we have choices and nargs as "+". For all other option, default behavior would be seen.